### PR TITLE
i18n for 2 .vue files

### PR DESCRIPTION
**Pull request by mcapodici**
Another commit for #127.

### CODE REVIEW

File: web/frontend/src/boot/i18n.ts [hunk 1]
```diff
@@ -3,14 +3,12 @@ import { createI18n } from 'vue-i18n';
 import messages from '../i18n';
```
Comment (mcapodici) [inline web/frontend/src/boot/i18n.ts:4]: The changes in this file are to help with allowing i18n from arbitrary functions, not just directly inside Vue components.

File: web/frontend/src/i18n/lang.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import { createI18nForApp } from 'src/boot/i18n';
```
Comment (mcapodici) [inline web/frontend/src/i18n/lang.ts:1]: might need a better name for this file!

File: web/frontend/src/pages/DroppedPinPage.vue [hunk 3]
```diff
@@ -34,7 +35,7 @@ async function loadDroppedPinPage(router: Router, position: LongLat) {
     return;
   }
   poi.value = {
-    name: 'Dropped Pin', // i18n
+    name: i18n('dropped_pin'),
```
Comment (mcapodici) [inline web/frontend/src/pages/DroppedPinPage.vue:38]: Tested this one manually and it works

File: web/frontend/src/pages/AlternatesPage.vue [hunk 4]
```diff
@@ -149,7 +149,7 @@ export default defineComponent({
           });
         },
         (error) => {
-          useQuasar().notify('Could not get GPS location'); // i18n
```
Comment (mcapodici) [inline web/frontend/src/pages/AlternatesPage.vue:152]: Couldn't get this to show up before the change, so untested by me.

File: web/frontend/src/pages/AlternatesPage.vue [hunk 5]
```diff
@@ -138,7 +138,7 @@ export default defineComponent({
       navigator.geolocation.getCurrentPosition(
         (position) => {
           fromPoi.value = {
-            name: 'My Location', // i18n
+            name: this.$t('my_location'),
```
Comment (mcapodici) [inline web/frontend/src/pages/AlternatesPage.vue:141]: Couldn't get this to show up before the change, so untested by me.

**Comment by ellenhp** [comment]
Do you know why the yarn.lock file is changing constantly? My understanding is that you are supposed to version those but it's kind of irritating that this is showing up as an 8k line diff.

**Comment by mcapodici** [comment]
Sorry didn't realise that was included in this PR. I am not using Yarn but NPM, which might be the reason. I'll start using it instead and that should solve it. For now I will revert the change.

**Review by ellenhp — APPROVED** [review]
Looks great! Thank you again for going and implementing this!

**Pull request merged**